app/src/main/java/com/team41/boromi/models/Book.java

```diff
@@ -12,10 +12,8 @@
  */
 public class Book implements Serializable {
 
```

Comment:
I might be wrong but doesn't a serializable class need a serializing tag ? I'm not too sure though since I don't have that much experience there. From https://www.baeldung.com/java-serialization
![image](https://user-images.githubusercontent.com/44931428/97526938-cd381100-196f-11eb-9d6b-27cb54349e3c.png)